scripts/storybook/webpack.config.js

```diff
@@ -2,22 +2,15 @@ const HardSourceWebpackPlugin = require('hard-source-webpack-plugin');
 const IgnoreNotFoundExportWebpackPlugin = require('ignore-not-found-export-webpack-plugin');
 const path = require('path');
 const getResolveAlias = require('../webpack/getResolveAlias');
+const webpack = require('webpack');
 
-module.exports = ({ config }) => {
-  config.module.rules.push({
-    test: /\.(ts|tsx)$/,
-    use: [
-      {
-        loader: require.resolve('ts-loader'),
-        options: {
-          transpileOnly: true,
-          experimentalWatchApi: true,
-          configFile: 'tsconfig.json',
-        },
-      },
-    ],
-  });
-
+/**
+ * @param {Object} param0
+ * @param {webpack.Configuration} param0.config
+ * @param {string} [param0.cwd] - cwd containing package.json resolve aliases should be based on
+ * (if different from actual cwd)
+ */
+function custom({ config, cwd }) {
```

Comment:
Aside from the addition of the cwd option, the changes to this file are just shuffling things around (not really adding anything)